src/Mod/Draft/TestDraft.py

```diff
@@ -108,6 +108,9 @@
 # from drafttests.test_oca import DraftOCA as DraftTest07
 # from drafttests.test_airfoildat import DraftAirfoilDAT as DraftTest08
 
+# Testing the utils module
+from drafttests.test_draftgeomutils import TestDraftGeomUtils as DraftTest09
```

Comment:
As this test deals with core functions and not importing of other file formats, I would add it to the sequence as `DraftTests04`, and move the rest of the tests after it.